src/content/certifications/exam-faqs/vault-associate-003.mdx

```diff
@@ -0,0 +1,123 @@
+<!-- Each "## Heading Two" section in this document will be used to populate an FAQ item -->
+
+## Exam Availability
+Coming soon!
+
+## Which Exam to Take
+You can use either exam to validate Vault knowledge at the associate level. The [Vault Associate 002 certification](/certifications/security-automation#vault-associate-002-details) is still relevant and will be accepted as validation of Vault knowledge until the badge’s expiration date. You can hold both the Vault Associate 002 and Vault Associate 003 at the same time.
```

Comment:
```suggestion
You can use either exam to validate Vault knowledge at the associate level. The [Vault Associate 002 certification](/certifications/security-automation#vault-associate-002-details) is still relevant and will be accepted as validation of Vault knowledge until the badge’s expiration date. You can hold both the Vault Associate 002 and Vault Associate 003 at the same time.

- Vault Associate 002: Exam available now
- Vault Associate 003: Coming early 2025. The Vault Associate 002 exam will no longer be available to take once Vault 003 is released.

```

Reply:
We can update this with specific dates and include "available for scheduling" as time goes on.